lib/src/geo/crs/crs.dart

```diff
@@ -104,7 +104,7 @@ abstract class Earth extends Crs {
   bool get infinite => false;
 
   @override
-  final Tuple2<double, double> wrapLng = const Tuple2(-180.0, 180.0);
+  final Tuple2<double, double> wrapLng = const Tuple2(-180, 180);
```

Comment:
This should be of type <int, int> shouldn't it? And I'm not sure how it would work but maybe final and const isn't needed - haven't suggested this though as I'm not 100% sure.

```suggestion
  final Tuple2<int, int> wrapLng = const Tuple2(-180, 180);
```

Reply:
I think this is wrong, they should be doubles because at the end LatLng still takes doubles. It's just a "cosmetic" change of not having unnecessary .0 enforce by the rule `prefer_int_literals`.